### PR TITLE
Fix client structs

### PR DESCRIPTION
**Pull request by artemreyt**

### CODE REVIEW

File: pkg/client/state_changes.go [hunk 1]
```diff
@@ -21,8 +21,10 @@ type IssueAction struct {
 	Name           string        `json:"name"`
 	Description    string        `json:"description"`
 	Decimals       int32         `json:"decimals"`
+	Quantity       int64         `json:"quantity"`
 	Reissuable     bool          `json:"isReissuable"`
-	CompiledScript []byte        `json:"compiledScript"`
+	CompiledScript *[]byte       `json:"compiledScript"` // optional
```
Comment (nickeskov) [inline pkg/client/state_changes.go:26]: Pointer here is unnecessary because zero value of any slice is `nil`.

**Review by nickeskov — CHANGES_REQUESTED** [review]

File: pkg/client/state_changes_test.go [hunk 2]
```diff
@@ -184,624 +217,65 @@ func TestUnmarshalStateChangesAllFields(t *testing.T) {
 	require.Equal(t, expectedStateChanges, actualChanges, "non equal stateChanges")
 }
 
-func TestName(t *testing.T) {
+func TestUnmarshalStateChangesLeaseCancelParams(t *testing.T) {
 	// mainnet tx 'HPsKrSJFFe9duhGbT3jb1YYnTninzmb99JRdRTtRefLw'
```
Comment (nickeskov) [inline pkg/client/state_changes_test.go:221]: Is this comment still relevant?

File: pkg/client/state_changes.go (hunk 1 above)
Comment (nickeskov) [reply pkg/client/state_changes.go:26]: Oh, I've realized, that yes. Closing.

File: pkg/client/state_changes_test.go (hunk 2 above)
Comment (nickeskov) [reply pkg/client/state_changes_test.go:221]: Oh, I've realized, that yes. Closing.

File: pkg/client/state_changes.go (hunk 1 above)
Comment (artemreyt) [reply pkg/client/state_changes.go]: done

**Review by nickeskov — APPROVED** [review]

**Review by alexeykiselev — APPROVED** [review]

**Pull request merged**